docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+
+services:
+  mysqldb:
+    container_name: mysqldb
+    image: mysql:latest
+    environment:
+      - MYSQL_ROOT_PASSWORD=LamborghiniLover
+      - MYSQL_DATABASE=rentmycar
+      - MYSQL_USER=user
+      - MYSQL_PASSWORD=LamborghiniLover
```

Comment:
Is er een manier waarop wij dit nog kunnen verbergen van GitHub?

Reply:
Ah ja deze repo is publiekelijk :D Had daar niet aan gedacht. Dit kunnen we verbergen met secrets. Zal dat nog implementeren

Reply:
Ah deze docker-compose.yml wordt niet gebruikt, maar heb deze gekopieerd naar mijn VPS. Dus ik kan deze uit Git halen, mara van de andere kant is een voorbeeld wel handig in git repo. Dus ik hernoem deze file naar example.docker-compose.yml.